implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/DynatraceMeterRegistry.java

```diff
@@ -185,11 +187,15 @@ private DynatraceCustomMetric createCustomMetric(Meter.Id id, long time, Number
     }
 
     private DynatraceCustomMetric createCustomMetric(Meter.Id id, long time, Number value, @Nullable DynatraceUnit unit) {
+        double doubleValue = value.doubleValue();
```

Comment:
Can we add unit tet for this behavior?

Reply:
@OleksiiBondar Sure. I thought so but I've been lazy as it was in the middle of the bed time 😅 

Thanks for pointing it out. I added tests in 76cc010.